src/lib/block/kuznyechik/kuznyechik.cpp

```diff
@@ -0,0 +1,4394 @@
+/*
+* GOST R 34.12-2015: Block Cipher "Kuznyechik"
+* RFC 7801
+*
+* This code is written by kerukuro for cppcrypto library (http://cppcrypto.sourceforge.net/)
+* and released into public domain.
+*
+* Botan is released under the Simplified BSD License (see license.txt)
+*/
+
+#include <botan/mem_ops.h>
+#include <botan/internal/kuznyechik.h>
```

Comment:
Include of `kuznyechik.h` should be the first include and separated by a newline from the remainder of the headers (so clang-format doesn't reorder it)

Reply:
Also done